types/codec.go

```diff
@@ -0,0 +1,27 @@
+package types
```

Comment:
You can remove this file altogether. This is amino-related, which has changed to protobuf

Reply:
Resolved